### DISCUSSION TITLE
I am getting out of video memory error when I try to run CarlaUE4.exe

### DISCUSSION THREAD

**Discussion by spacebot29** [carla-simulator/carla#4671] [Q&A]
![image](https://user-images.githubusercontent.com/36999515/134364616-a4006b2a-f5fd-4926-b69e-e85c67865118.png)

My current version is Carla 0.9.12. I have 8GB RAM and 4 GB RTX 3050 GPU. I have seen people run Carla at far lower configuration.

I have restarted my pc and closed all apps. Still getting this error.

Thanks in advance.

**Comment by mjpablo23** [comment]
hi, did you resolve this?  I am trying to decide between getting a computer with a rtx 3060 vs 3070.  thanks.

**Reply by aterzgar** [reply]
CarlaUE4 -dx11 you can try this command. it solved my problem.

**Reply by KnownR** [reply]
which version did you use?????

**Comment by spacebot29** [comment]
Yes, Carla 0.9.12 (latest version) has very high graphic requirements
(possibly 16 GB 3070) and extensive installation with several dependencies
on Unreal Engine.I was not able to run it. But the earlier versions (0.9.8
and earlier) can run smoothly on older systems (4GB vRAM). Also, the
installation is straight forward, just copy the github repository.

**Comment by mjpablo23** [comment]
ah ok thanks.  did you need to build it, or just installed it?

**Comment by spacebot29** [comment]
Just copy the whole repository and you'll be fine. Also, check that you are
using certain version of python. it's either 3.6 or 3.7.

**Reply by mjpablo23** [reply]
ah ok thanks.  do you have windows or linux?  I downloaded a zip file and extracted it on a windows machine.

**Comment by spacebot29** [comment]
Windows

**Comment by spacebot29** [comment]
I don't remember anything about it.

But I'll just share you my previous replies long ago:

*> Yes, Carla 0.9.12 (latest version) has very high graphic requirements
(possibly 16 GB 3070) and extensive installation with several dependencies
on Unreal Engine.I was not able to run it. But the earlier versions (0.9.8
and earlier) can run smoothly on older systems (4GB vRAM). Also, the
installation is straight forward, just copy the github repository*.

> ah ok thanks. did you need to build it, or just installed it?

> Just copy the whole repository and you'll be fine. Also, check that you
are using certain version of python. it's either 3.6 or 3.7. For windows.
